configs/openmrs_config/roles/cambodia_roles.csv

```diff
@@ -1,2 +1,4 @@
+8c26a98b-fa03-4c37-b788-6d1e537d4071,NCD care provider in Health Center,NCD care provider in Health Center,NCD care provider,View NCD Screening encounters;Edit NCD Screening encounters;View NCD Medical History encounters;View NCD Consultation encounters,
+ffba7d63-c1cf-470f-8a58-ad393c42fc98,NCD care provider in Hospital,NCD care provider in Hospital,NCD care provider,View NCD Medical History encounters;Edit NCD Medical History encounters;View NCD Consultation encounters;Edit NCD Consultation encounters;Vew NCD Screening encounters,
```

Comment:
@kdaud, do you need to add the `View` and `Edit` privileges if those are part of the associated inherited roles?

Reply:
@Ruhanga have omitted  them.

Reply:
You didn't have to remove them. Sorry I didn't mean to mislead you this way. https://github.com/openmrs/ozone-distro-cambodia/pull/103/commits/9594cfb409e7291e1895bb48020c5ea611f18623 is not necessary. I just wanted you to be sure you know what you are doing. :-)

Reply:
> I just wanted you to be sure you know what you are doing. :-)

I knew these specific privileges were required for the respective added roles. However, your [comment](https://github.com/openmrs/ozone-distro-cambodia/pull/103#discussion_r1181636561) made me cast a vote of dought.

Reply:
@kdaud , this privilege is not created. See the typo here: `Vew NCD Screening encounters`